downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -172,4 +151,28 @@ object Git : GitBase() {
             throw DownloadException("Could not clone $vcsUrl.", e)
         }
     }
+
+    private fun getCommittish(vcsRevision: String?, packageVersion: String, targetDir: File): String {
+        if (vcsRevision != null && vcsRevision.isNotEmpty())
```

Comment:
Style nit: We prefer to use curly branches even for single line blocks.

Reply:
Done.  too many different code bases. spaces, tabs, 2, 4, semi-colon, no, ...

I also managed to get the CLA thing sorted. Not really sure what happened but my user settings were borked on my machine and ...

The Appveyor `branch` status check does not seem to be happy though. Similarly on my other PR